src/app/courses/add-courses/courses-add.component.ts

```diff
@@ -69,7 +71,8 @@ export class CoursesAddComponent implements OnInit {
       method: '',
       gradeLevel: this.gradeLevels[0],
       subjectLevel: this.subjectLevels[0],
-      createdDate: Date.now()
+      createdDate: Date.now(),
+      creator: this.userService.get().name
```

Comment:
once we have course shared between planets it will not be valid as same username can be taken by another user on different planet